src/components/events-card/events-card.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Container,
+  Title,
+  Description,
+  Image,
+  Content,
+  DetailsLine,
+} from './events-card.style';
+import { EventsButton } from '../events-button';
+
+interface EventsCardProps {
+  image: string;
+  title: string;
+  description: string;
+  value: number;
+  date: string;
+  link: string;
+}
+
+export const EventsCard = ({
+  image,
+  title,
+  description,
+  value,
+  date,
+  link,
+}: EventsCardProps): React.ReactElement => {
+  return (
+    <Container>
+      <Image image={image} />
+
+      <Content>
+        <div>
+          <Title>{title}</Title>
+
+          <Description>{description}</Description>
+        </div>
+
+        <div>
+          <DetailsLine>
+            <p>
+              {value === 0
+                ? 'Gratuito'
+                : Intl.NumberFormat('pt-BR', {
+                    style: 'currency',
+                    currency: 'BRL',
+                  }).format(value)}
+            </p>
+            <p>{date}</p>
+          </DetailsLine>
+
+          <div style={{ display: 'flex', justifyContent: 'center' }}>
+            <EventsButton
+              link={link}
+              ariaLabel={`Botão para se inscrever no evento ${title}`}
```

Comment:
Aqui você pode colocar "inscreva-se no evento x"
Se for um botão msm o leitor de tela fala o tipo de elemento que ele é. Se não for um botão, vc pode usar o atributo role="button" que funciona do msm jeito! Se quiser conversar um pouquinho sobre leitores de tela, me dá um alô ❤